database/sequelize.js

```diff
@@ -0,0 +1,36 @@
+const {Sequelize, DataTypes} = require('sequelize');
+const InfoModel = require('../models/info.js');
+const Info_photosModel = require('../models/Info_photos.js');
+const Characteristics_dataModel = require('../models/Characteristics_data.js');
+const Characteristics_reviewsModel = require('../models/Characteristics_reviews.js');
+
+const sequelize = new Sequelize('reviews_api', 'jkum1', '123', {
+  host: 'localhost',
+  dialect: 'postgres',
+  define: {
+    timestamps: false,
+    freezeTableName: true,
+  }
+});
+
+const Info = InfoModel(sequelize, Sequelize);
+const Info_photos = Info_photosModel(sequelize, Sequelize);
+const Characteristics_data = Characteristics_dataModel(sequelize, Sequelize);
+const Characteristics_reviews = Characteristics_reviewsModel(sequelize, Sequelize);
+
+sequelize.authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+sequelize.sync();
+
+module.exports = {
```

Comment:
Probably just for refactoring, but would it be clearer to create the model instances and import them within their own files they are defined?

Reply:
White space/newline in some areas would improve readability. Possibly putting some of the larger code chunks within functions to help describe what is happening or the purpose of the code blocks.